src/QueueMessageExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Text;
+using Azure.Storage.Queues.Models;
+
+namespace Likvido.Worker.AzureStorageQueue
+{
+    public static class QueueMessageExtensions
+    {
+        public static string EncocodeMessageText(this string message)
```

Comment:
Is this method used anywhere?

Reply:
I've added this method to be used via consumers. To match `GetDecodedMessageText`

Reply:
You mean it should be used by producers, right?

So, the idea is that you base64 encode the message before pushing it to a queue?

Reply:
Yes, in long term I'd say we should create another package something like "Likvido.Utils"